src/esp/assets/ResourceManager.h

```diff
@@ -1256,7 +1256,7 @@ class ResourceManager {
    * being rendered, since that mesh will have its components moved into actual
    * render mesh constructs.
    */
-  std::unique_ptr<GenericSemanticMeshData> infoSemanticMeshData_{};
+  std::unique_ptr<GenericSemanticMeshData> infoSemanticMeshData_;
```

Comment:
This is one of the funnier STL workarounds I came across lately.

Without the `{}`, the type can be forward-declared and it will work as long as all constructors, assignment operators and destructors are deinlined in the file. With the `{}` however, a GCC 7 build will attempt to instantiate `std::default_delete<GenericSemanticMeshData>` each time the header is included anywhere, failing if the type definition is not known.